drivers/SCsub

```diff
@@ -26,9 +26,15 @@ if env["xaudio2"]:
         print_error("Target platform '{}' does not support the XAudio2 audio driver".format(env["platform"]))
         Exit(255)
     SConscript("xaudio2/SCsub")
+
 # Shared Apple platform drivers
 if env["platform"] in ["macos", "ios"]:
     SConscript("apple/SCsub")
+
+# Accessibility
+if env["platform"] in ["macos", "windows", "linuxbsd"]:
```

Comment:
I'd check `env["accesskit"] and env["platform"] in ["macos", "windows", "linuxbsd"]` here, and remove the `if` check from `accesskit/SCsub`.